### DISCUSSION TITLE
Lack of transparency around slurs const in reserved.ts

### DISCUSSION THREAD

**Discussion by ixtli** [bluesky-social/atproto#1325] [General Discussion]
It is unclear why english words well regarded as slurs were removed by pr ( https://github.com/bluesky-social/atproto/pull/1319 ) without any discussion. I am curious how, where, and when the decision to merge PRs like #1319 are made because this is something that, as a user, should have community participation.

**Comment by mletterle** [comment]
A broader discussion on Trust and Safety in general is needed as well, this is just one example.  Glad to see anything being done, but the communication so far hasn't been... great.

**Comment by snarfed** [comment]
https://github.com/bluesky-social/atproto/discussions/1329

**Reply by ixtli** [reply]
thank you for linking this, ill read it :) 

i'd like to also restate that this post was really just meant to get some clarification about what process exists to merge changes to the configurations for the techniques and systems described in #1329

**Comment by Scotchester** [comment]
With the release of https://github.com/bluesky-social/atproto/pull/1336, I don't expect we'll ever get any clarity on why the removals in this PR were made, or what the true list is going forward. Not exactly confidence-inspiring.

**Reply by Scotchester** [reply]
Acknowledging that several flavors of explanation have now been given, and I take them at their word, though I still wish they had come much quicker.
